src/components/form/OneClickForm/components/DataField/DataFieldHeader/HeaderSelect/index.tsx

```diff
@@ -30,24 +23,18 @@ export function HeaderSelect() {
   const inputRef = useRef<HTMLDivElement | undefined>(undefined);
   const [inputWidth, setInputWidth] = useState<string | undefined>(undefined);
 
-  const boxStyle: Record<string, SxProps> | undefined = useMemo(() => {
-    const isTallerBox = ['AddressCredential'].includes(
-      credentialDisplayInfo.credentialRequest?.type ?? 'unknown',
-    );
-
-    if (isTallerBox) {
-      return {
-        '& div[role="combobox"]': {
-          width: '100%',
-          height: 'auto',
-          overflow: 'hidden',
-          whiteSpace: 'unset !important',
-        },
-      };
-    }
-
-    return undefined;
-  }, [credentialDisplayInfo.credentialRequest?.type]);
+  // Replace from instances list the actual display info, so all changes that happens there reflects into the select UI.
```

Comment:
Thanks for this inline comment here... would have a very hard time understand what is going on otherwise